### PR TITLE
Adds Nightly package for `keras-cv`

### PR DESCRIPTION
**Pull request by sampathweb**
Adds Nightly package for `keras-cv`

### CODE REVIEW

**Comment by divyashreepathihalli** [comment]
@sampathweb Thank you for the PR! LGTM but I would wait till our tests are passing with the new keras 3 compat changes before releasing nightlies.

**Comment by sampathweb** [comment]
> @sampathweb Thank you for the PR! LGTM but I would wait till our tests are passing with the new keras 3 compat changes before releasing nightlies.

Sure.  We can wait to merge.

File: pip_build.py [hunk 1]
```diff
@@ -49,14 +50,34 @@ def ignore_files(_, filenames):
     return [f for f in filenames if "test" in f]
 
 
-def build():
+def export_version_string(version, is_nightly=False):
```
Comment (mattdangerw) [inline pip_build.py:53]: It looks like our version string in `__init__.py` is the current release version. Maybe we should update that to `0.7.0` so it is pointing to the next major version we will release?

**Review by mattdangerw — APPROVED** [review]
looks good to me thanks! just one comment

File: pip_build.py (hunk 1 above)
Comment (sampathweb) [reply pip_build.py:53]: Done

**Comment by sampathweb** [comment]
I will merge this PR now.  Plan to fix the tests before nightly runs tonight.

**Pull request merged**